source/SIL.AppBuilder.Portal.Frontend/package.json

```diff
@@ -44,7 +44,7 @@
     "redux-saga": "^0.16.0",
     "reflux": "^6.4.1",
     "sass-loader": "^7.0.3",
-    "semantic-ui-css": "^2.3.2",
+    "semantic-ui-css": "^2.3.3",
```

Comment:
Check if Headers dropdown works because I had to revert a preview version because of that

Reply:
yup, seems fine, can't open both at the same time.